db/migrate/20230323110801_create_likes.rb

```diff
@@ -1,8 +1,6 @@
 class CreateLikes < ActiveRecord::Migration[7.0]
   def change
     create_table :likes do |t|
-      t.references :user, null: false, foreign_key: true
-      t.references :post, null: false, foreign_key: true
 
       t.timestamps
     end
```

Comment:
- [ ] Great job but it's one of the project requirements you Create custom methods that your User, Post, Comments, and Likes models should include. See the screenshot below 👇🏻 Kindly add this ✍🏻 and make sure to check if your methods are working well from your rails console 💯 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/227429262-5cd85bba-2888-464b-bf41-70c5cdec9259.png)